### ISSUE TITLE
TypeError: 'NoneType' for empty args.always_use

### ISSUE THREAD

**Issue by kbiethahn** [ARandomOWL/liberty-to-genlib#6]
Hi,

so testing the script on the hdll_tt_025C liberty file from the skywater PDK, I encountered the error
```TypeError: 'NoneType' object is not iterable```.
I tried it with liberty-parser 0.0.8 first, then went back to 0.0.7 but the error stayed the same. Since I am pretty new to all this I will just show you the command + trace:

```
python3 liberty-to-genlib.py ~/openlane/pdks/skywater-pdk/libraries/sky130_fd_sc_hdll/latest/timing/sky130_fd_sc_hdll__tt_025C_1v80.lib > workcraft_hdll_tt_025C.lib
Traceback (most recent call last):
  File "liberty-to-genlib.py", line 124, in <module>
    rs = map(re.compile, flatten(args.always_use))
  File "liberty-to-genlib.py", line 123, in <lambda>
    flatten = lambda t: [item for sublist in t for item in sublist]
TypeError: 'NoneType' object is not iterable
```

Have you tested it on other liberty files, so that it might be an issue with the format from my PDK? I am trying to get a library to use with Workcraft and it is meant for a QDI design. Therefore timing issues are not critical.

It would be great, if you could point me in the right direction. Thanks!

**Comment by ARandomOWL** [comment]
The error is unrelated to the SKywater PDK and is caused by empty `args.always_use`, because nothing was passed to `--always-use` on the command line. This is unintended behaviour and I will fix ASAP.

Following on, there seems to be another problem with `EscapedString` type. I will investigate and open a new issue.

**Comment by ARandomOWL** [comment]
Fixed by 69cef6c5c623a5ed6bf2e856dcfb858db8d83bc0. See #7

**Comment by ARandomOWL** [comment]
@kbiethahn I have fixed a couple of things now. The tool has run successfully on the default Skywater lib and the output looks reasonable. Please try again and open a new issue for any other problems!
